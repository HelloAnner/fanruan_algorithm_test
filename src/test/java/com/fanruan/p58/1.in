1 2 1
2 3 10
5 10 11
6 11 2