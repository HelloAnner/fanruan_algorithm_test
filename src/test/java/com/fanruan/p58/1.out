1 3 11
5 11 13